(/guides/add-onboarding-flow) Add code comment that explains user.reload() (#1212)
clerk/clerk-docs: docs/guides/add-onboarding-flow.mdx

- A process in your frontend with logic to collect and submit all the information for onboarding. In this guide, you will create a simple form.
- A method in your backend to securely update the user's `publicMetadata`

> > [!IMPORTANT]
> > After the backend call returns successfully, call `user.reload()` on the frontend to refresh the **signed-in** user's session token. This ensures that all future requests made from the user include the updated `publicMetadata`.
> 

Reviewer: ```suggestion
<Callout type="info">
 After the backend call returns successfully, call `user.reload()` on the frontend to refresh the **signed-in** user's session token. This ensures that all future requests made from the user include the updated `publicMetadata`.
</Callout>
```

> > [!IMPORTANT]
> > After the backend call returns successfully, call `user.reload()` on the frontend to refresh the **signed-in** user's session token. This ensures that all future requests made from the user include the updated `publicMetadata`.
> 

Reviewer 2: ```suggestion
```

1. In your `/onboarding` directory, create a `page.tsx` file.
2. Add the following code to the file.

> ```tsx {{ mark: [16, 17] }}

Reviewer 2: ```suggestion
```tsx
```

  const handleSubmit = async (formData: FormData) => {
    const res = await completeOnboarding(formData);
    if (res?.message) {

>       // Reload user to get updated User object

Reviewer 2: ```suggestion
      // Reloads the user's data from Clerk's API
```

  const handleSubmit = async (formData: FormData) => {
    const res = await completeOnboarding(formData);
    if (res?.message) {

>       // Call reload() to refresh the user's session token 

Reviewer: Not sure this comment is necessary with the new Callout section
```suggestion
```

Reviewer 2: I'd prefer a comment than a callout

Author: Done!

>       // Call reload() to refresh the user's session token 

Reviewer 2: ```suggestion
      // Reload user to get updated User object
```

Reviewer: I defer to you :)
